reverb/cc/platform/default/repo.bzl

```diff
@@ -332,6 +349,12 @@ def _reverb_protoc_archive(ctx):
     urls = [
         "https://github.com/protocolbuffers/protobuf/releases/download/v%s/protoc-%s-linux-x86_64.zip" % (version, version),
     ]
+    if is_darwin(ctx):
+        urls = [
+            "https://github.com/protocolbuffers/protobuf/releases/download/v%s/protoc-%s-osx-x86_64.zip" % (version, version),
+        ]
+        sha256 = ""  # TODO(Feiteng) set this in WORKSPACE
```

Comment:
we need this todo done in this PR since it's a security issue to not have a sha256.

Reply:
Is there a simple way set `PROTOC_SHA256` in `WORKSPACE` according to platform?